tests/deployment/aws_lambda/test_aws_lambda_deployment_operator.py

```diff
@@ -83,7 +83,6 @@ def get_service_api(self, name):
 
     def mock_lambda_app(func):
         @mock_s3
-        @patch('bentoml.utils.s3.download_directory_from_s3', return_value=None)
```

Comment:
@yubozhao looks like these tests were outdated? could you take a look and make sure those unit tests and mocks still make sense